tests/ui/derives/issue-91550.stderr

```diff
@@ -94,12 +82,7 @@ error[E0599]: the method `use_ord_and_partial_ord` exists for struct `Object<NoD
   --> $DIR/issue-91550.rs:28:9
    |
 LL | pub struct NoDerives;
-   | --------------------
-   | |
-   | doesn't satisfy `NoDerives: Eq`
-   | doesn't satisfy `NoDerives: Ord`
-   | doesn't satisfy `NoDerives: PartialEq`
-   | doesn't satisfy `NoDerives: PartialOrd`
+   | -------------------- doesn't satisfy `NoDerives: Eq`, `NoDerives: Ord`, `NoDerives: PartialEq` or `NoDerives: PartialOrd`
```

Comment:
would be nice to have a single "`NoDerives` does not implement `Eq`, `Ord`, `PartialEq` and `PartialOrd`".

Reply:
I'd rather we did that as part of its own PR to make reviewing easier :)

Reply:
yea makes sense